Formula/kmc.rb

```diff
@@ -0,0 +1,32 @@
+class Kmc < Formula
+  # cite Deorowicz_2015: "https://doi.org/10.1093/bioinformatics/btv022"
+  desc "Fast and frugal disk based k-mer counter"
+  homepage "http://sun.aei.polsl.pl/kmc/"
+  url "https://github.com/marekkokot/KMC/archive/v3.0.1.tar.gz"
+  sha256 "0dbc9254f95541a060d94076d2aa03bb57eb2da114895848f65af0db1e4f8b67"
+  head "https://github.com/marekkokot/KMC.git"
+
+  depends_on "gcc" # needs OpenMP
+
+  fails_with :clang do
+    build 600
```

Comment:
Remove this line. It fails with all versions of `clang`. This line says that versions older than 600 fail, but the current version of `clang` is fine.

Reply:
I originally had removed it, but put it back in in desparation. Removed now.